lib/stm32/f4/rcc.c

```diff
@@ -307,6 +357,22 @@ const struct rcc_clock_scale rcc_hse_25mhz_3v3[RCC_CLOCK_3V3_END] = {
 		.apb1_frequency = 42000000,
 		.apb2_frequency = 84000000,
 	},
+    { /* 96MHz */
+		.pllm = 25,
+		.plln = 192,
```

Comment:
What's the idea here? HSI is at 16MHz, VCO input must be between 1 and 2Mhz, and is recommended to be 2Mhz to reduce jitter.  16/25 is < 1 MHz.  It looks sort of like you copied this from a 25MHz crystal config?